cmd/influxd/run.go

```diff
@@ -42,14 +42,36 @@ func NewRunCommand() *RunCommand {
 }
 
 type Node struct {
-	broker   *influxdb.Broker
-	dataNode *influxdb.Server
+	Broker   *influxdb.Broker
+	DataNode *influxdb.Server
 	raftLog  *raft.Log
+
+	adminServer      *admin.Server
+	clusterListener  net.Listener // The cluster TCP listener
+	snapshotListener net.Listener
 }
 
 func (s *Node) Close() {
-	if s.broker != nil {
-		if err := s.broker.Close(); err != nil {
+	if err := s.closeClusterListener(); err != nil {
+		log.Fatalf("error closing cluster listener: %s", err)
```

Comment:
I wonder if we should log that we couldn't close, and at the end, if anyone couldn't close, then we fatal.  At least try to cleanly shut down what we can.  It probably doesn't matter now, but might matter in the future if we cleanly stop http requests, to routines, etc.